lib/lbt/bundle/AutoSplitter.js

```diff
@@ -180,6 +199,15 @@ class AutoSplitter {
 					totalSize += 21 + toRequireJSName(module).length;
 				});
 				break;
+			case SectionType.DepCache:
+				currentSection = {
+					name: section.name,
+					mode: SectionType.DepCache,
+					filters: section.modules.slice()
+				};
+				currentModule.sections.push( currentSection );
+				totalSize += moduleSizes["__depCacheSize"];
```

Comment:
The use of a fixed, single entry in `moduleSizes` limits us to a single `depCache` section and does not allow to split that section, if it is huge. (one section in the bundle definition could be split into multiple `sap.ui.loader.config` calls in the final bundle).

Reply:
Thanks, @codeworrior !
If I understand you correctly, this would mean that we can split the depCache definitions into different files, like the preload does. So, in the end all those definintions `sap.ui.loader.config({depCacheUI5:{}});` would be merged and not override themselves, is this correct?